metaworld/envs/mujoco/sawyer_xyz/sawyer_xyz_env.py

```diff
@@ -458,11 +458,9 @@ def sawyer_observation_space(self):
     def step(self, action):
         assert len(action) == 4, f"Actions should be size 4, got {len(action)}"
         self.set_xyz_action(action[:3])
+        if self.curr_path_length == self.max_path_length:
```

Comment:
Should this be greater than or equal?

Reply:
@pseudo-rnd-thoughts yeah that's fair. My thinking was that it *should* be impossible to ever get past the max path length but I can make that change